### PR TITLE
Add dev.qwery.AddWater

### PR DESCRIPTION
**Pull request by largestgithubuseronearth**
### Please confirm your submission meets all the criteria

- [X] Please describe the application briefly.
A GUI utility app to conveniently download the Firefox GNOME Theme, install it to your Firefox profile, and automatically update it when you log in.

- [X] The domain used for the application ID is [controlled by the application developers][appid-domain] and the [application id guidelines][appid] are followed.
- [X] I have read the and followed all the [Submission and licence requirements][reqs].
- [X] I have [built][build] and tested the submission locally.
- [X] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about this submission. **Link:**

[appid-domain]: https://docs.flathub.org/docs/for-app-authors/requirements/#control-over-domain-or-repository
[appid]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id
[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission

### CODE REVIEW

File: dev.qwery.AddWater.json [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+                    "url" : "https://github.com/largestgithubuseronearth/addwater",
+                    "tag" : "rc1.1"
+                }
+            ],
+            "builddir" : true
+        }
+    ],
+    "build-options" : {
+        "env" : {        }
+    }
```
Comment (bbhtt) [inline dev.qwery.AddWater.json:101]:
```suggestion
    ]
```

File: dev.qwery.AddWater.json [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    "command" : "addwater",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--filesystem=~/.mozilla/firefox/:rw",
+        "--filesystem=~/.var/app/org.mozilla.firefox/.mozilla/firefox/:rw",
+        "--filesystem=~/.var/app/org.mozilla.firefox/.mozilla/Firefox/:rw",
```
Comment (bbhtt) [inline dev.qwery.AddWater.json:15]: The second is doesn't seem to exist. it's `firefox` in lowercase.

Comment (largestgithubuseronearth) [reply dev.qwery.AddWater.json:15]: I was concerned they may change it in the future given Thunderbird's ID _is_ uppercase. But I'll remove it for now.

**Comment by bbhtt** [comment]
bot, build dev.qwery.AddWater

**Pull request closed without merging**